### DISCUSSION TITLE
Conversion of generated path into json file format

### DISCUSSION THREAD

**Discussion by Botronics** [Fields2Cover/Fields2Cover#58] [General]
Inside the farm. Im generating swath path using fields2cover package. Generated swath path inside the farm  has to be converted to json file format.

**Comment by Gonzalo-Mier** [comment]
Hi @Botronics,
This functionality is missing yet, and it would be a good improvement of the repo.
For now, I suggest using the function F2CPath::saveToFile(const std::string& name_file) that saves the path into a CSV. Then convert the CSV to Json with any other program.

Either way I agree it would be a nice PR :)
